.eslintrc.js

```diff
@@ -6,4 +6,8 @@ module.exports = {
   },
   extends: ['@smg-automotive/eslint-config'],
   parser: '@typescript-eslint/parser',
+  rules: {
+    'no-shadow': 'off',
```

Comment:
you cannot export enums when this rule is on. It was replaced by the ⬇️ 

see: https://github.com/typescript-eslint/typescript-eslint/issues/2471

Reply:
Let's incorporate it into our global config then. Not a blocker, can be done in a follow-up.